### PR TITLE
[Publisher][Admin Panel] Fit 3 team member cards in a row

### LINKED ISSUE #1560: [Publisher][Admin Panel] Team Member & Roles Tab: Fit 3 team member cards in a row
Within the Team Member & Roles Tab in the Agency Provisioning modal, the UI only fits 2 cards per row if the viewport width is less than approx. 1710px - which is too wide of a breakpoint. It would be ideal if we could fit 3 team member cards until we hit ~1400 pixels (roughly). We can accomplish this with a combination of reducing the card width and increasing the modal size.

### PR DESCRIPTION
**Pull request by nasaownsky**
Tweak team members card and modal sizes to fit cards better in smaller screens

## Type of change

> All pull requests must have at least one of the following labels applied (otherwise the PR will fail):

| Label                       	| Description                                                                                               	|
|-----------------------------	|-----------------------------------------------------------------------------------------------------------	|
| Type: Bug                   	| non-breaking change that fixes an issue                                                                   	|
| Type: Feature               	| non-breaking change that adds functionality                                                               	|
| Type: Breaking Change       	| fix or feature that would cause existing functionality to not work as expected                            	|
| Type: Non-breaking refactor 	| change addresses some tech debt item or prepares for a later change, but does not change functionality    	|
| Type: Configuration Change  	| adjusts configuration to achieve some end related to functionality, development, performance, or security 	|
| Type: Dependency Upgrade      | upgrades a project dependency - these changes are not included in release notes                             |

## Related issues

closes #1560 

### Development

**This box MUST be checked by the submitter prior to merging**:
- [ ] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by nichelle-hall** [comment]
I don’t know enough about the mechanics of fitting three cards on the screen to provide feedback on that aspect. However, on my MacBook Pro 16-inch in full-screen mode, it looks like there’s room for up to four cards to fit. I’m leaving the PR open for Mahmoud to review when he gets back.

Screenshot 2024-11-26 at 10 49 33 AM

**Comment by nasaownsky** [comment]
> I don’t know enough about the mechanics of fitting three cards on the screen to provide feedback on that aspect. However, on my MacBook Pro 16-inch in full-screen mode, it looks like there’s room for up to four cards to fit.

Hmm, strange -- on the same system it fits 4 cards, I specifically tweaked it this way. I'll return to testing again.

**Comment by nasaownsky** [comment]
I update fitting method -- now it should be much smoother and should match the requirement of 3 cards per row in 1400px-ish widths. @mxosman @nichelle-hall

File: publisher/src/components/AdminPanel/AdminPanel.styles.tsx [hunk 1]
```diff
@@ -334,8 +334,9 @@ export const ButtonWrapper = styled.div`
 `;
 
 export const TeamMembersContainer = styled.div`
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
```
Comment (mxosman) [inline publisher/src/components/AdminPanel/AdminPanel.styles.tsx:338]: Oh nice! Always forget we can just do it this way. 👏

**Review by mxosman — APPROVED** [review]
Ah - love this - feels very smooth! Thank you for your eyes & suggestion, @nichelle-hall and for the elegant solution, @nasaownsky! 😎

**Pull request merged**